src/sphinx_book_theme/_transforms.py

```diff
@@ -0,0 +1,51 @@
+from sphinx.transforms.post_transforms import SphinxPostTransform
+from typing import Any
+from docutils import nodes as docutil_nodes
+from .nodes import SideNoteNode
+
+
+class HandleFootnoteTransform(SphinxPostTransform):
+
+    default_priority = 1
+    formats = ("html",)
+
+    def run(self, **kwargs: Any) -> None:
+        theme_options = self.env.config.html_theme_options
+        if theme_options.get("use_sidenotes", False) is True:
```

Comment:
To remove an extra indendation, could this instead be:

```python
if theme_options.get("use_sidenotes", False) is False:
    return None
...and then continue the logic w/o indentation here
```

Reply:
sure thing. Done in https://github.com/executablebooks/sphinx-book-theme/pull/546/commits/bb485b6ccbce37083a0df073f1a174e380b3a756